README.md

```diff
@@ -503,34 +221,25 @@ Accepted log types:
 	1. To test CLI changes locally you can run `node <project_dir>/src/bin/medic-conf.js`. This will run as if you installed via npm.
 	1. To test changes that are imported in code run `npm install <project_dir>` to use the local version of medic-conf.
 
-## compress images
-
-To compress PNGs and SVGs in the current directory and its subdirectories, two commands are available:
-
-	compress-pngs
-	compress-svgs
-
-## Wishlist
-
-* only upload things which have changed (this could be a separate mode - e.g. `update` vs `configure`)
-
 # Releasing
 
-## Do I need to update `release-notes.md`?
-
-As we strive to have clear, readable commit messages for every change, [release-notes.md](./release-notes.md) should not be updated for every single change. Instead, it should be used as a forum for deep information about significant changes, breaking changes, changes to interfaces, changes in behavior, new feature details, etc.
-
-## Performing the release
-
 1. Create a pull request with prep for the new release. This should contain changes to release notes if required and anything else that needs to be done:
 1. Get it reviewed and approved
 1. Run `npm version patch`, `npm version minor`, or `npm version major` as appropriate. This will:
-  - Update versions in `package.json` and `package-lock.json`
-  - Commit those changes locally and tag that commit with the new version
-  - "Compile" and publish the changes to npm
+    - Update versions in `package.json` and `package-lock.json`
+    - Commit those changes locally and tag that commit with the new version
+    - "Compile" and publish the changes to npm
 1. `git push && git push --tags` to push the npm generated commit and tag up to your pre-approved pull request
 1. Merge the pull request back into master
 
+Note that as we strive to have clear, readable commit messages for every change, [release-notes.md](./release-notes.md) should not be updated for every single change. Instead, it should be used as a forum for deep information about significant changes, breaking changes, changes to interfaces, changes in behavior, new feature details, etc.
+
+# Build Status
```

Comment:
I would complain about this because I prefer the build badges to be at the top, but as we've done the same in cht-core at least I suppose we're consistent?

Reply:
Ya, I just did it for consistency but also think that having the build status at the bottom is fine since it matters for comparatively fewer people (there are more users of the tool than devs of it).